python/scientific-python.ipynb

```diff
@@ -515,7 +515,7 @@
     "from scipy.signal import find_peaks\n",
     "peaks, _ = find_peaks(x, distance=150)\n",
     "\n",
-    "fig, ax = plt.subplots(1, 1, constrained_layout=True)\n",
+    "fig, ax = plt.subplots(1, 1, layout="constrained")\n",
```

Comment:
Did you edit this in the notebook file? Because now the json is broken....

Reply:
Need to escape the " inside the string: `\"`

Reply:
Oh... I see.
seems like notebooks don't like sed commands :D 
I'll fix it ASAP